### ISSUE TITLE
Error with db implementation

### ISSUE THREAD

**Issue by ASKemp** [fingers10/JqueryDataTablesServerSide#12]
Hi,
I'm trying to implement your code over a database example. I've made minimum changes to the example to instead point to a db table.
When I debug I get an error with the GetDataAsync function, Visual Studio is reporting the error as...
"Incorrect syntax near 'OFFSET'. Invalid usage of the option NEXT in the FETCH statement"

I'm trying against a MS SQL database.

**Comment by fingers10** [comment]
Hi ASKemp,

Can you share you Model code? do you have default sort enabled?

Cheers,
Abdul

**Comment by ASKemp** [comment]
Hi, thanks for the quick reply.
I've literally taken your code and...
changed the Program.cs to be default (as you get with a default mvc .net app) and to remove the seed data etc.
Changed the startup.cs to reference my db connection instead
services.AddDbContext<Fingers10DbContext>(options => options
                   .UseSqlServer(_configuration.GetConnectionString("DefaultConnection")));

I've changed the demo.cs and demoentity.cs to reference a table in the database, just two fields in the view model for testing.

Changed the app.js to reference just the two fields.
Everything else is the same really, apart from the error when the GetDataAsync executes the query - it brings back the record count okay.

EDIT: Just looked up some documentation, can see that OFFSET NEXT and FETCH are SQL Servier 12 onwards whereas my test database is running on SQL Server 2008 R2 - so I'm guessing that may be the problem?
